From: Lena Voytek <lena@voytek.dev>
Date: Tue, 28 Apr 2026 22:39:51 +0800
Subject: Update default datadir config to /var/lib/mariadb.

Last-Update: 2026-03-24

Use /var/lib/mariadb as the default datadir for Debian/Ubuntu to improve
co-installability between MySQL and MariaDB servers.
Last-Update: 2026-03-24

Forwarded: https://github.com/MariaDB/server/pull/5059
---
 cmake/install_layout.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/install_layout.cmake b/cmake/install_layout.cmake
index d52836a..c46bac3 100644
--- a/cmake/install_layout.cmake
+++ b/cmake/install_layout.cmake
@@ -192,7 +192,7 @@ SET(INSTALL_MYSQLTESTDIR_DEB            "share/mariadb/mariadb-test")
 SET(INSTALL_SQLBENCHDIR_DEB             ".")
 SET(INSTALL_SUPPORTFILESDIR_DEB         "share/mariadb")
 #
-SET(INSTALL_MYSQLDATADIR_DEB            "/var/lib/mysql")
+SET(INSTALL_MYSQLDATADIR_DEB            "/var/lib/mariadb")
 
 SET(INSTALL_RUNDATADIR_DEB              "/run/mysqld")
 SET(INSTALL_UNIX_ADDRDIR_DEB            "${INSTALL_RUNDATADIR_DEB}/mysqld.sock")
